packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -744,6 +744,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
       {overlayImageItem && (
         <ImageOverlay
           {...overlayImageItem}
+          altText={'Image Details Opened'}
```

Comment:
It probably needs to be localizable

Reply:
I can remove this actually, thanks for catching.

Reply:
You could maybe also just re-use the aria-label here too